src/sentry/snuba/metrics/naming_layer/mapping.py

```diff
@@ -82,6 +82,14 @@ def get_public_name_from_mri(internal_name: Union[TransactionMRI, SessionMRI, st
         raise InvalidParams(f"Unable to find a mri reverse mapping for '{internal_name}'.")
 
 
+def is_mri_private(internal_name: Union[TransactionMRI, SessionMRI, str]) -> bool:
+    try:
+        get_public_name_from_mri(internal_name)
```

Comment:
do you have test for this function? I would prefer to add single one.

Reply:
I mean unit test for is_private_mri

Reply:
I had this in mind, will do!